### PR TITLE
Created Login page in react and added JWT Token authentication

### PR DESCRIPTION
**Pull request by praduman03**
Initialized and implemented JWT token for authorization and access
Implemented a Login page for Frontend part of the project.

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ import ReactDOM from 'react-dom/client';
...
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import {BrowserRouter} from 'react-router-dom'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
+  
   <React.StrictMode>
+    <BrowserRouter>
     <App />
```
Comment (aditya-190) [inline frontend/src/index.js:13]: Use Eslint for clean code.

File: frontend/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```
Comment (aditya-190) [inline frontend/README.md:4]: Update your Readme file to include a basic description about your project.

File: frontend/src/App.js [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
-        >
-          Learn React
-        </a>
-      </header>
+      {/* <Link to='/'>Home</Link>
+      <Link to='/Explore'>Explore</Link> */}
+
+      <Routes>
+      <Route path='/' element={<Home/>}/>
+      <Route path='/Explore' element={<Explore/>}/>
```
Comment (aditya-190) [inline frontend/src/App.js:15]: Try to keep the routes in lowercase.

File: node_modules/.bin/loose-envify [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
```
Comment (aditya-190) [inline node_modules/.bin/loose-envify]: Please add node_modules in .gitignore.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make the required changes. Everything else LGTM

File: node_modules/.bin/loose-envify (hunk 4 above)
Comment (praduman03) [reply node_modules/.bin/loose-envify]: Added node_modules in gitignore and deleted it from the repo

**Pull request merged**